test/zapYearnShatesTest.ts

```diff
@@ -0,0 +1,102 @@
+import { expect } from "chai";
```

Comment:
you might want to reconsider this filename `test/zapYearnShatesTest.ts`.   I think you meant `test/zapYearnSharesTest.ts`